### DISCUSSION TITLE
select and reject filter with "search"

### DISCUSSION THREAD

**Discussion by hklages** [pallets/jinja#1779] [Q&A] [answered]
Hi. 

In Home Assistant I use the following Jinja statement to count items of a csv (with separator //) containing the string "Sabotage". It works fine but I can not find any documentation about "search" in select. 
In particular I would like to know the meaning of the 3rd parameter - in this case "false". 

```
 {{ states('sensor.raspberrymatic_sv_myservicemessages').split('//') | select('ne','') 
      | select('search','Sabotage ', false) | list | count | int(default=0) }}
```

**Comment by davidism** [comment, marked as answer]
It's a test provided by Home Assistant, not a built-in to Jinja. You'll need to look at their documentation about it.
